<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCarWindow</class>
 <widget class="QDialog" name="EditCarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Car</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/res/data/icons/edit.webp</normaloff>:/res/data/icons/edit.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#EditCarWindow
{
	color:333;
}

QLabel#enterIDlbl, QLabel#enterINewName, QLabel#enterINewBrand, QLabel#enterNewPrice, QLabel#enterCarDescription
{
	color:white;
	font-size:12px;
	background-color:#222;
	border-radius:4px;
}

QLabel#changeCarlbl
{
	font-size:20px;
	color:white;
	background-color:#222;
	border-radius:4px;
}

QFrame#frameEditCar
{
	background-color:#333;
}

QPushButton#changeCarButt
{
	color:white;
	background-color:#111;
	font-size:13px;

}

QPushButton#changeCarButt::hover
{
	color:white;
	font-size:13px;
	background-color:#444;
}

QTextEdit#TextEditNewDescrpt
{
	background-color:#222;
}

QLineEdit#IDLineEdit, QLineEdit#CarNewNameLineEdit, QLineEdit#CarNewBrandLineEdit, QLineEdit#NewPricelbl
{
	background-color:#222;
}</string>
  </property>
  <widget class="QLabel" name="enterIDlbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  Enter the ID of the car you wish to change :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="enterINewName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  Enter the Cars new Model Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CarNewNameLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="enterINewBrand">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  Enter the Cars new Brand Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CarNewBrandLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="enterNewPrice">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  Enter the Car's new Price :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPricelbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="enterCarDescription">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  Enter the Car's new description :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextEditNewDescrpt">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeCarButt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Car</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameEditCar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>621</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="changeCarlbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  Change Car</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-size:10px;
</string>
    </property>
    <property name="text">
     <string>Leave blank if unchanged</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>frameEditCar</zorder>
  <zorder>enterIDlbl</zorder>
  <zorder>IDLineEdit</zorder>
  <zorder>enterINewName</zorder>
  <zorder>CarNewNameLineEdit</zorder>
  <zorder>enterINewBrand</zorder>
  <zorder>CarNewBrandLineEdit</zorder>
  <zorder>enterNewPrice</zorder>
  <zorder>NewPricelbl</zorder>
  <zorder>enterCarDescription</zorder>
  <zorder>TextEditNewDescrpt</zorder>
  <zorder>changeCarButt</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
